OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -100,28 +135,44 @@ $ docker container start -i WCPAdminContainer
 Run the following command to create the Portal Managed Server container:
 
 ```
-$ docker run -i -t --name WCPortalContainer --network=WCPortalNET -p <Any Free Port>:$MANAGED_SERVER_PORT -v $DATA_VOLUME:/u01/oracle/user_projects --env-file <directory>/webcenter.env.list $WCPortalImageName configureOrStartWebCenterPortal.sh
+$ docker run -i -t --name <WCP_CONTAINER_NAME> --network=<NETWORK_NAME> -p <HostFreePort>:<MANAGED_SERVER_PORT> -v <DATA_VOLUME>:/u01/oracle/user_projects --env-file webcenter.env.list oracle/wcportal:12.2.1.4 configureOrStartWebCenterPortal.sh
+
+```
+Sample Run Command 
+```
+$ docker run -i -t --name WCPortalContainer --network=WCPortalNET -p 8888:8888 -v /scratch/wcpdocker/volumes/wcpportal:/u01/oracle/user_projects --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4 configureOrStartWebCenterPortal.sh
 
 ```
+WebCenter Portal   Container start up command explained:
+
+| 		Parameter    	   |     Parameter Name             | 							     		Description			                               |
+| :----------------------: | :-----------------------------:| :---------------------------------------------------------------------------------------: |
+| --name                   | WCP_CONTAINER_NAME                 |  Set to WebCenter Portal Server Container name                                                |
+| --network                | NETWORK_NAME                   | User-defined network to connect to; use the one created earlier ‘WCPortalNET’.             |
+| -p                       | HostFreePort:MANAGED_SERVER_PORT|  Set Managed Server port ,Maps the container port to host's port.              	   |
+| -v                      | DATA_VOLUME                   | Mounts the host directory as a Volume.      |
+| --env-file               | webcenter.env.list  | `webcenter.env.list` sets the environment variables.                            |
+|oracle/wcportal:12.2.1.4| REPOSITORY:TAG             | Repository name:Tag name of the image created using buildDockerImage.sh 
```

Comment:
Remove "create using buildDockerImage.sh" and perhaps try, "The (optional) repository, name and tag of the image"